mPower2/MotorControl/ViewControllers/MCTOverviewStepViewController.swift

```diff
@@ -173,6 +173,30 @@ open class MCTOverviewStepViewController : RSDOverviewStepViewController {
         }
     }
     
+    /// Override goForward to add in requesting permission to access the motion sensors before continuing.
+    /// This is done here b/c the researchers wanted the balance and walk to use separate recorders. This
+    /// means that the recorder isn't turned on (and thus asking for permission) until after the user has
+    /// put the device in their pocket. syoung 07/25/2018
```

Comment:
FWIW, on (and *only* on) a participant's first run of the walk/balance task, after the walk part it's supposed to tell them to take the phone out of their pocket and look at the instructions before continuing to the balance part.

You'd still want to do this here though I suppose, in case they turned off permissions later.

Reply:
Permission turned off is handled already in the base class. This is bc the first time through, the recorder doesn’t turn on for “walk” unitl after the phone is in the pocket. (Only while walking and not pre/post)